src/obfl-specification.html

```diff
@@ -1320,6 +1320,29 @@ <h4 id="L836" class="include">page-number</h4>
 </dl>
 </div>
 
+<h4 id="L840" class="include">external-reference</h4>
+
+<ins datetime="20200429T15:00:00">The external-reference element can be attached to any position in the text within the block
+  to send information through the dotify framework from obfl to pef.</ins>
```

Comment:
Hmm, this might seem nitpicking, but it feels wrong to mention Dotify here. There are no references to Dotify in the whole document. Replace it with for instance "user agent" or "implementing application" (see the "General Terms" section).

Same story for PEF, although here it might actually help to explain what the feature is about. But then there should be a reference in the "References" section at the bottom.

What about the following:

> With the external-reference element custom annotations can be attached to any position in the text within a block or field. How the user agent processes the information is implementation dependent.
> 
> ### Example
> 
> A typical use case is to pass information through unchanged from the source document to the destination, for instance a PEF document [PEF]. The result document is annotated with the exact same information attached to the corresponding positions in the braille.
> 
> _then some example XML in- and output:_
>
> ```xml
> <obfl> ... </obfl>
> ```
> ```xml
> <pef> ... </pef>
> ```
>
> _and below, under References_:
>
> **[PEF]**
>     "[PEF 1.0 - Portable Embosser Format (Public draft)](https://mtmse.github.io/pef/pef-specification.html)", Joel Håkansson, 12 September 2011.
>     [Latest version](https://mtmse.github.io/pef/) available at: https://mtmse.github.io/pef

Reply:
Hi @bertfrees

Well, writing a specification requires some nitpicking :)

I've added the reference in the specification and some examples and changed to the wording you suggested. It seemed like a really good suggestion.

Best regards
Daniel